aip/general/0008.md

```diff
@@ -114,18 +114,33 @@ followed by a bulleted list explaining the example.
 Individual subsections can be cited individually, and further elaborate
 details.
 
+## Rationale
+
+The "rationale" section is optional, and helps the reader understand the
+motivation for specific guidance within the AIP.
+
+Deeper explanations of design justification and tradeoffs **must** be in the
+rationale instead of other sections, to ensure the rest of the document acts as
+an easily digestible reference.
+
+## History
+
+The "history" section is optional, and documents events and context around
+a significant edit (e.g. rewrite) of an AIP.
```

Comment:
This is an AIP, let's not use e.g.

> ...a significant edit to an AIP. For example, explanation of rewrite would be included in this section

Reply:
sure, happy to do it. Do we have guidance to not include abbreviations in AIPs?